internal/cmd/ske/kubeconfig/create/create.go

```diff
@@ -0,0 +1,150 @@
+	skeUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/ske/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/ske"
+)
+
+const (
+	clusterNameArg = "CLUSTER_NAME"
+
+	expirationFlag = "expiration"
+	locationFlag   = "location"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ClusterName    string
+	Location       *string
+	ExpirationTime *string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("create %s", clusterNameArg),
+		Short: "Creates a kubeconfig for an SKE cluster",
+		Long: fmt.Sprintf("%s\n%s",
+			"Creates a kubeconfig for a STACKIT Kubernetes Engine (SKE) cluster.",
+			"By default the kubeconfig is created in the .kube folder, in the user's home directory. The kubeconfig file will be overwritten if it already exists."),
+		Args: args.SingleArg(clusterNameArg, nil),
+		Example: examples.Build(
+			examples.NewExample(
+				`Create a kubeconfig for the SKE cluster with name "my-cluster"`,
+				"$ stackit ske kubeconfig create my-cluster"),
+			examples.NewExample(
+				`Create a kubeconfig for the SKE cluster with name "my-cluster" and set the expiration time to 30 days`,
+				"$ stackit ske kubeconfig create my-cluster --expiration 30d"),
+			examples.NewExample(
+				`Create a kubeconfig for the SKE cluster with name "my-cluster" and set the expiration time to 2 months`,
+				"$ stackit ske kubeconfig create my-cluster --expiration 2M"),
+			examples.NewExample(
+				`Create a kubeconfig for the SKE cluster with name "my-cluster" in a custom location`,
+				"$ stackit ske kubeconfig create my-cluster --location /path/to/config"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create a kubeconfig for SKE cluster %q? This will OVERWRITE your current configuration, if it exists.", model.ClusterName)
+				err = confirm.PromptForConfirmation(cmd, prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req, err := buildRequest(ctx, model, apiClient)
+			if err != nil {
+				return fmt.Errorf("build kubeconfig create request: %w", err)
+			}
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("create kubeconfig for SKE cluster: %w", err)
+			}
+
+			// Create the config file
+			if resp.Kubeconfig == nil {
+				return fmt.Errorf("no kubeconfig returned from the API")
+			}
+
+			if model.Location == nil {
+				kubeconfigPath, err := skeUtils.GetDefaultKubeconfigLocation()
+				if err != nil {
+					return fmt.Errorf("get default kubeconfig location: %w", err)
+				}
+				model.Location = &kubeconfigPath
+			}
```

Comment:
Could you write this to another variable? Just to avoi modifying the input model

Something like
```go
configLocation := model.Location
if configLocation == nil {
    kubeconfigPath, err := skeUtils.GetDefaultKubeconfigLocation()
    if err != nil {
        return fmt.Errorf("get default kubeconfig location: %w", err)
    }
    configLocation = &kubeconfigPath
}
```